README.md

```diff
@@ -63,6 +63,11 @@ Requires Ruby 2.3.3 or higher. macOS and Xcode are required when building iOS pr
 bundle install
 git submodule init
 git submodule update
+brew update && brew upgrade
```

Comment:
`brew upgrade` upgrades all installed tools, we probably shouldn't suggest this since it might break something.